### PR TITLE
Rename Uniques Error::Unknown to something more sensible

### PR DESCRIPTION
**Pull request by gavofyork**

### CODE REVIEW

File: frame/uniques/src/functions.rs [hunk 1]
```diff
@@ -131,7 +131,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		ensure!(!Asset::<T, I>::contains_key(class, instance), Error::<T, I>::AlreadyExists);
 
 		Class::<T, I>::try_mutate(&class, |maybe_class_details| -> DispatchResult {
-			let class_details = maybe_class_details.as_mut().ok_or(Error::<T, I>::Unknown)?;
+			let class_details = maybe_class_details.as_mut().ok_or(Error::<T, I>::UnknownClassClass)?;
```
Comment (ggwpez) [inline frame/uniques/src/functions.rs:134]:
```suggestion
			let class_details = maybe_class_details.as_mut().ok_or(Error::<T, I>::UnknownClass)?;
```

File: frame/uniques/src/functions.rs [hunk 2]
```diff
@@ -165,9 +165,9 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		let owner = Class::<T, I>::try_mutate(
 			&class,
 			|maybe_class_details| -> Result<T::AccountId, DispatchError> {
-				let class_details = maybe_class_details.as_mut().ok_or(Error::<T, I>::Unknown)?;
+				let class_details = maybe_class_details.as_mut().ok_or(Error::<T, I>::UnknownClassClass)?;
```
Comment (ggwpez) [inline frame/uniques/src/functions.rs:168]:
```suggestion
				let class_details = maybe_class_details.as_mut().ok_or(Error::<T, I>::UnknownClass)?;
```

**Review by ggwpez — COMMENTED** [review]
Typos  

Btw. can I directly push such changes or does it cause conflicts in your work flow?

**Comment by gavofyork** [comment]
Go ahead and push 👍

**Review by ggwpez — APPROVED** [review]

**Review by KiChjang — APPROVED** [review]

**Comment by KiChjang** [comment]
bot merge

**Pull request merged**